pytorch_lightning/callbacks/gradient_accumulation_scheduler.py

```diff
@@ -15,8 +15,8 @@ class GradientAccumulationScheduler(Callback):
 
     Args:
         scheduling: scheduling in format {epoch: accumulation_factor}
-            .. warning:: Epochs indexing starts from "1" until v0.6.x,
-                but will start from "0" in v0.8.0.
+        .. warning:: Epochs indexing starts from "1" until v0.6.x,
```

Comment:
I think this was on purpose because the warning belongs to the argument...

Reply:
ok, but changing the indentation removed the error message

Reply:
maybe warnings for arguments need to be done in another way

Reply:
nope, fixed it! thanks